POS_Image-RPi-Bootstrap/rpi-bootstrap-7.0.0/config.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<image schemaversion="6.1" name="POS_Image_RPi-Bootstrap">
+    <description type="system">
+        <author>Admin User</author>
+        <contact>noemail@example.com</contact>
+        <specification>Bootable Raspberry Pi SD Card image with Saltboot initrd</specification>
+    </description>
+    <preferences>
+        <version>7.0.0</version>
+        <packagemanager>zypper</packagemanager>
+        <bootsplash-theme>SLE</bootsplash-theme>
+        <bootloader-theme>SLE</bootloader-theme>
+
+        <locale>en_US</locale>
+        <keytable>us.map.gz</keytable>
+        <timezone>Europe/Berlin</timezone>
+        <hwclock>utc</hwclock>
+
+        <rpm-excludedocs>true</rpm-excludedocs>
+        <type filesystem="ext4" image="oem" initrd_system="dracut" firmware="efi" bootpartition="false" efipartsize="64" editbootinstall="editbootinstall_rpi.sh"></type>
+    </preferences>
+
+    <packages type="image" patternType="onlyRequired">
+        <package name="curl"/> 
+        <package name="dracut"/>
+        
+        <package name="grub2-branding-SLE" bootinclude="true"/>
+        <package name="grub2"/>
+        <package name="grub2-arm64-efi" arch="aarch64"/>
+        <package name="haveged"/>
+        <package name="parted"/>
+        <package name="wicked"/>
+
+        <package name="kernel-default"/>
+	<package name="salt-minion"/>
+
+	<package name="dracut-saltboot"/>
+        <package name="mdadm"/>
+        <package name="cryptsetup"/>
+        <package name="kexec-tools"/>
+
+        <package name="plymouth"/>
+        <package name="plymouth-dracut"/>
+        <package name="plymouth-branding-SLE"/>
+        <package name="plymouth-plugin-label-ft"/>
+
+        <package name="fontconfig"/>
+        <package name="fonts-config"/>
+        <package name="noto-sans-fonts"/>
```

Comment:
This is needed for displayed text on plymouth splash, right? 32MiB.. :(

Reply:
yes

Reply:
Do we need to use exactly this font or any truetype? What about [Cantarell](https://fonts.google.com/specimen/Cantarell). I see that this is available on SLES and package is less then 1MiB.
```
Information for package cantarell-fonts:
----------------------------------------
Repository     : os_pool_repo
Name           : cantarell-fonts
Version        : 0.111-1.31
Arch           : noarch
Vendor         : SUSE LLC <https://www.suse.com/>
Support Level  : Level 2
Installed Size : 717.1 KiB
Installed      : Yes (automatically)
Status         : up-to-date
Source package : cantarell-fonts-0.111-1.31.src
Summary        : Contemporary Humanist Sans Serif Font
```